src/Microsoft.AspNetCore.Hosting/Internal/HostingApplication.cs

```diff
@@ -64,21 +74,65 @@ public Context CreateContext(IFeatureCollection contextFeatures)
                 // Non-inline
                 LogRequestStarting(httpContext);
             }
-
             if (diagnoticsEnabled)
```

Comment:
In other places we have another set of events and checks:
* IsEnabled("*.Activity") used as a instrumentation switch 
* "*.Activity.Start" - test for start and event name
* "*.Activity.Stop" - stop event (no test)

However there is an intention to use Begin/End events to keep things the same for existing consumers. If this is the case, one of them has to be used